TODO.txt

```diff
@@ -21,3 +21,8 @@ formulas into editor as the user types is a complicated due to the way these edi
 
 On every change of cursor, check if the cursor is on a timestamp mark. If so, toggle away the
 timestamp mark from currently activated timestamp.
+
+
+#### Extension
+
+If user clicks the extension button more than once, there's multiple inject scripts running. Each does the same work per message - like capturing the same frame and sending it to app. This needs to be fixed.
```

Comment:
Could it be made so that user doesn't have to click? And only one extension is registered.

Reply:
I'm guessing some coordination is required between the app and the extension. Since the script *does* need to be loaded every time a new video is selected. But not more than once while in one `/editor/...` page. Looking into it.

Reply:
Ok. Probably something which is triggered everytime a new youtube frame is loaded. Along with a unique id for the frame, so both can communicate?

Reply:
Going to need two inject scripts for this. One that does the usual frame capture work, and one that gets loaded only once and communicates between the app and the background script.

Reply:
How about using videoId, which is unique. Use videoId to check if already injected, if not inject script.

Reply:
Lets work on the injection fix on a separate branch. In this branch, could you move the files to `src/` and copy then to `dist/` at build time.